apps/chat-e2e/src/tests/parametrizedReplay.test.ts

```diff
@@ -285,7 +285,7 @@ dialTest(
           chatBar.importButton.click(),
         );
         await conversations.openEntityDropdownMenu(conversation.name);
-        await conversations.selectEntityMenuOption(MenuOptions.replay, {
+        await conversationDropdownMenu.selectMenuOption(MenuOptions.replay, {
```

Comment:
Why do we have a separate method for selectDuplicateMenuOption, but do not use the same approach here?

Reply:
'Duplicate' option is available for both side-bar entities - conversation and prompt, that's why I created a common method selectDuplicateMenuOption() in SideBarEntities class. However, 'Replay', 'Playback'etc options are defined only for conversations.

Reply:
removed selectDuplicateMenuOption and replaced it with selectMenuOption() call